app/jobs/ingest_mets_job.rb

```diff
@@ -120,7 +120,11 @@ def slug_to_id(slug)
     # Provide the resource class used for ingesting Resources using a METS Document
     # @return [Class]
     def resource_klass
-      ScannedResource
+      if import_mods?
```

Comment:
So this allows us to have a source_metadata_identifier in a simple resource, but not import metadata from there? like a pulfa object?

Reply:
This is assuming we don't have MODS resources with a source_metadata_identifier in the METS.